packages/schematics/migrations/update-8-0-0/update-8-0-0.spec.ts

```diff
@@ -0,0 +1,353 @@
+      `);
+    });
+
+    it(`should be migrated from '@nrwl/schematics' to '@nrwl/angular'`, async () => {
+      initialTree.create(
+        'file.ts',
+        `
+        import * from '@nrwl/schematics/src/utils/fileutils';
+        import { fileExists } from '@nrwl/schematics/src/utils/fileutils';
+      `
+      );
+
+      const tree = await schematicRunner
+        .runSchematicAsync('update-8.0.0', {}, initialTree)
+        .toPromise();
+      expect(tree.readContent('file.ts')).toEqual(
+        `
+        import * from '@nrwl/workspace/src/utils/fileutils';
+        import { fileExists } from '@nrwl/workspace/src/utils/fileutils';
+      `
+      );
+    });
+  });
+
+  describe('builders', () => {
+    it('should be migrated', async () => {
+      const tree = await schematicRunner
+        .runSchematicAsync('update-8.0.0', {}, initialTree)
+        .toPromise();
+      const { projects } = readJsonInTree(tree, 'angular.json');
+      const { architect } = projects['my-app'];
+      expect(architect.cypress.builder).toEqual('@nrwl/cypress:cypress');
+      expect(architect.jest.builder).toEqual('@nrwl/jest:jest');
+      expect(architect.nodeBuild.builder).toEqual('@nrwl/node:build');
+      expect(architect.nodeServe.builder).toEqual('@nrwl/node:execute');
+      expect(architect.webBuild.builder).toEqual('@nrwl/web:build');
+      expect(architect.webServe.builder).toEqual('@nrwl/web:dev-server');
+      expect(architect.runCommands.builder).toEqual(
+        '@nrwl/workspace:run-commands'
+      );
+    });
+  });
+
+  describe('update npm script', () => {
+    it('should do ng update @nrwl/workspace', async () => {
+      const tree = await schematicRunner
+        .runSchematicAsync('update-8.0.0', {}, initialTree)
+        .toPromise();
+      const packageJson = readJsonInTree(tree, 'package.json');
+      expect(packageJson.scripts.update).toEqual('ng update @nrwl/workspace');
+    });
+  });
+
+  describe('jest config', () => {
+    it('should have the plugin path migrated', async () => {
+      initialTree.create(
+        'jest.config.js',
+        `
+        module.exports = {
+          resolver: '@nrwl/builders/plugins/jest/resolver',
+        };
+      `
+      );
+      const tree = await schematicRunner
+        .runSchematicAsync('update-8.0.0', {}, initialTree)
+        .toPromise();
+      expect(tree.readContent('jest.config.js')).toContain(
+        '@nrwl/jest/plugins/resolver'
+      );
+    });
+  });
+
+  describe('dependencies', () => {
+    it('should change to the new dependencies', async () => {
+      const tree = await schematicRunner
+        .runSchematicAsync('update-8.0.0', {}, initialTree)
+        .toPromise();
+
+      const { dependencies, devDependencies } = readJsonInTree(
+        tree,
+        'package.json'
+      );
+      expect(dependencies['@nrwl/nx']).not.toBeDefined();
+      expect(devDependencies['@nrwl/schematics']).not.toBeDefined();
```

Comment:
Do you think it makes sense to keep this one in case folks have external schematics? 

Maybe at least we should print a warning message saying: either add @nrwl/schematics or update your external schematics?